h/static/scripts/annotation-controller.coffee

```diff
@@ -0,0 +1,38 @@
+angular = require('angular')
+
+
+module.exports = class AnnotationController
```

Comment:
I don't think renaming this to AnnotationController is a great idea. We now have two distinct objects called AnnotationController. Let's keep this as AnnotationViewerController for now, even if these two things need unifying later.